keyboards/projectcain/vault45/readme.md

```diff
@@ -0,0 +1,19 @@
+# vault45
+
+![vault45](https://imgur.com/a/ShWl4h8)
+
+*This is a 40% HHKB-style keyboard inspired by the JD45 with updated bottom row options and rotary encoders.*
+
+* Keyboard Maintainer: [projectcain](https://github.com/projectcain)
+* Hardware Supported: *Vault 45 PCB*
+* Hardware Availability: *mechvault.net*
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader?

Reply:
added as directed